sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -209,12 +208,12 @@ client.create(chatThread: thread) { result, _ in
 
 #### Get a thread
 
-Use the `getChatThread` method to retrieve a thread.
+Use the `get` method of `ChatClient` to retrieve a thread.
 
 - `chatThreadId` is the unique ID of the thread.
 
 ```swift
-client.getChatThread(chatThreadId: threadId) { result, _ in
+chatClient.get(chatThreadId: threadId) { result, _ in
```

Comment:
I'm a little torn on this. When we have a method whose name is only a verb, the first parameter must be the noun that the verb operates on. This method "gets" a thread, not a thread ID, so calling it `get(threadId:)` doesn't make sense. It could be called `get(thread:)`, if it were obvious to the customer that the `thread` parameter is in fact the thread ID. However I am a little dubious that this would be the case.

An alternative could be `getThread(withId:)` which moves the noun (back) into the method name, and provide some nice consistency with `listThreads` below. I am leaning towards this as the best option unless you feel that a customer would immediately understand what they are supposed to provide as the parameter to `get(thread:)`.

Reply:
Also just as a note, if we went with the former option, the internal name for the parameter could still denote that we're asking for the thread ID here (that is, the API signature in source code would be something like `get(thread threadId: String)`). I'm not sure if the customer would actually see the internal name though to help guide them.

Reply:
I don't necessarily think `client.get(thread: String, ...)` is unintuitive. In Storage we have `get(container: String, ...)` which is a name. I don't think the customer would ever assume that the noun referred to the strong object type, since if you had that object, you wouldn't need to `get` it.

Reply:
As discussed offline I've update to follow `get(thread: String)`, using internal names where appropriate. There will be some exceptions in the case of the list methods as well as sendReadReceipt and sendTypingNotification